integration-tests/integration_test.go

```diff
@@ -18,9 +18,10 @@ import (
 	"github.com/Nerzal/gocloak/v13"
 	"github.com/cenkalti/backoff/v4"
 	"github.com/cucumber/godog"
-	"github.com/nexodus-io/nexodus/internal/cucumber"
+    "github.com/nexodus-io/nexodus/internal/cucumber"
```

Comment:
appears to be some broken whitespace changes here and elsewhere

Reply:
`make generate` should fix this.